wikidatarefisland/data_model/filters.py

```diff
@@ -28,3 +28,28 @@ def get_property_id_statement_includer(included_properties):
         that are any of the passed property ids
         """
         return lambda statement: statement.get('pid', '') in included_properties
+
+
+class ItemFilters:
+    @staticmethod
+    def get_item_class_excluder(excluded_classes):
+        """
+        Returns a filter that is false for all items in the provided class
```

Comment:
PEP257, you should remove "s" from "Returns" and either keep all of it to one line or have a subject line and have multiple line docstring.

Reply:
Done! Multiline in place